public/src/js/script.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * 
+ * Displays information about the method been called.
+ * @param {String} id - id of method to display
+ * @returns {void}
+ */
+const showMethodDetail = (id) => {
+  $('.details pre div').map((index, domElement) => {
+    domElement.style.display = 'none';
+    if(id) {
+      const attr = domElement.getAttribute('id');
+      if(id === attr) {
+        domElement.style.display = 'block';
+      }
+    }
+  });
+};
+
+/**
+ * Runs the method selected.
+ * @returns {void}
+ */
+const runMethod = () => {
+  let result = null;
+  const methodCalled = $('#string-method').val();
+  let value = $('#string-value').val(); 
+  if(methodCalled && value.length > 0) {
+    if(methodCalled in String.prototype) {
+      $('.details').css('display', 'block');
+      if(['isDigit', 'toCurrency'].includes(methodCalled)) {
+        if(isNaN(parseInt(value))) {
```

Comment:
why not use a regex for this considering the fact that parseInt(`123string`) will return 123
and also for `toCurrency`, I may want to convert 123.45 so the word `integer` may not be so appropriate in the error message